src/pages/index.mdx

```diff
@@ -4,23 +4,69 @@ import { SiteLayout } from "../components/layout/SiteLayout";
 
 <SiteLayout>
 
-<h1 className="text-center font-display">11th Annual oSTEM Conference 2021</h1>
-
-<ConferenceCountdown />
-
-<img src="/images/2021/podium-presenters.jpg" alt="" className="rounded-3xl p-3"/>
-
-<p className="text-center text-3xl font-display">Your seat at our table</p>
-<p className="text-center text-3xl font-display">
-  October 28 &ndash; 31, 2021, Virtual
-</p>
-
-<div className="text-2xl flex justify-center p-8">
-  <CTAButton onClick="/2021/rfp">Submit a Proposal</CTAButton>
+<div className="border-b-4 border-mulberry border-solid pb-4">
+    <p className="flex text-center text-5xl font-display font-black text-mulberry">11th Annual oSTEM Conference 2021</p>
+    <p className="flex justify-center text-2xl font-display italic">Your seat at our table</p>
```

Comment:
Can we change the title to 'oSTEM 2021 11th Annual Conference' as this is consistent with 2020 and 2019?

Can we title case the theme: 'Your Seat at Our Table'?